compiler/rustc_codegen_llvm/src/debuginfo/metadata.rs

```diff
@@ -87,8 +75,6 @@ type SmallVec<T> = smallvec::SmallVec<[T; 16]>;
 mod enums;
 mod type_map;
 
-pub(crate) use type_map::TypeMap;
```

Comment:
I moved this to where the other imports are, so that it doesn't trick rust-analyzer into adding new imports in the wrong place.